shell/platform/embedder/tests/embedder_unittests_gl.cc

```diff
@@ -2281,6 +2310,7 @@ TEST_F(EmbedderTest,
           FlutterBackingStore backing_store = *layers[0]->backing_store;
           backing_store.type = kFlutterBackingStoreTypeOpenGL;
           backing_store.did_update = true;
+          backing_store.is_cacheable = true;
```

Comment:
We need another test case where a frame is being rendered at the same size (there are already tests that do this) and where we should assert that the engine is indeed discarding layers with backing stores of identical size.